### ISSUE TITLE
MapBox on Simulator extremely slow on Xcode 10

### ISSUE THREAD

**Issue by racer1988** [mapbox/mapbox-gl-native-ios#172]
Hello! 

I noticed similar issues opened for Xcode 9 (like mapbox/mapbox-gl-native#10199).
I nearly can't use the map in the simulator.

I would like to know if this is a known issue, some settings I need to configure or something else.

I included the map in a view controller:

```
class MapViewController: UIViewController {

private lazy var mapView: MGLMapView = {
        let mapView = MGLMapView(frame: view.bounds, styleURL: mapBoxUrl)
        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
        // Set the map view's delegate
        mapView.delegate = self
        view.addSubview(mapView)
        return mapView
    }()
}
```

and the entire controller containing the map is included in another controller via child-parent connection:

```
func add(child: UIViewController, to parent: UIViewController) {
        parent.addChild(child)
        parent.view.addSubview(child.view)
        child.didMove(toParent: parent)
    }


extension ContainerViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        add(child: mapViewController, to: self)
        
        
    }
}
```

### Configuration

* Mapbox SDK version: **Mapbox-iOS-SDK (4.9.0)** 
* iOS version: **12.1** 
* macOS version: **10.14.3 (18D109)** 
* Device/simulator models: **iPhone XR** 
* Xcode version: **Version 10.1 (10B61)** 

Thanks

**Comment by raho** [comment]
I'm also experiencing poor performance on the simulator. Using Xcode 11.4 
In the logs I see:  GPU Identifier: Apple Software Renderer

**Comment by kajensen** [comment]
I'm seeing this issue with the window manager app Magnet (mentioned above). Odd bug.. will this be fixed?

**Comment by julianrex** [comment]
This was fixed by https://github.com/mapbox/mapbox-gl-native-ios/pull/318, which was included in the iOS `6.0.0` SDK.
